docs/error_messages.md

```diff
@@ -0,0 +1,368 @@
+# Error Counters
+
+Issues found in the input files are categorized under "counters". Counters
+aggregate the issues, and provide a high-level overview of what went wrong.
+
+Error counters with "**Suffix Description:**" are prefix counters that are
```

Comment:
Counters with "Suffix Description" have an additional suffix that details the subject of the error